README.md

```diff
@@ -129,6 +129,41 @@ You can pull your queries directly from a standard GitHub search! If you want to
 
 As a CLI, `good-first-issue` uses the Commander.js CLI framework. If you want to better understand how our CLI is built, commander.js is pretty [well documented](https://github.com/tj/commander.js/). Also used are Chalk for terminal coloring and [boxen](https://github.com/sindresorhus/boxen) to simplify the output container implementation.
 
+## Release Process
+
+Good First Issue follows a relatively strict release process intended to ensure the spice flows.
+
+### Versioning
+
+| Semantic Version | Type | Reason |
+|-------------------|-------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| Major (**x**.x.x) | Breaking changes and non-trivial upgrades | Ensuring that end-users can rely on Good First Issue not breaking however they're consuming it |
+| Minor (x.**x**.x) | Project additions, other feature additions | Following the SemVer standard, project additions and feature additions are backwards-compatible enhancements. We generally try to ship one addition per Minor. |
+| Patch (x.x.**x**) | Bug fixes, minor enhancements to metadata and content | Tiny, hardly visible fixes to improve UX/DX or fix the module |
+
+## Labels and Milestones
+We use both GitHub Labels and Milestones to track releases. Since project additons count as a minor release, we prefer to space those out and ship them individually rather than shipping many at once. This pace may be revised later, but for now it introduces the need for a release queue and setting things up to be released ahead of them acutally being released.
```

Comment:
```suggestion
We use both GitHub Labels and Milestones to track releases. Since project additions count as a minor release, we prefer to space those out and ship them individually rather than shipping many at once. This pace may be revised later, but for now, it introduces the need for a release queue and setting things up to be released ahead of them actually being released.
```
Grammerly helped me here!